charts/argo-cd/values.yaml

```diff
@@ -1898,6 +1898,7 @@ server:
     # -- Enable an ingress resource for the Argo CD server
     enabled: false
     # -- Additional ingress annotations
+    ## Annotation values may reference other values by {{ .Values.otherValue }} (be sure protect by quotation marks)
```

Comment:
I get what you're trying to say here, but I'm not sure it's super-clear for newer users. Could you re-write with an inline example maybe?

Reply:
@tico24 Hi
I added extended description to README file and added reference from value.yaml:
https://github.com/argoproj/argo-helm/pull/2306/files#diff-9b2b8960b2091a28395a78f3f4b76c4310b39062f6602f9a0575a27ed890f904R1228